.github/actions/bump_precommit_hook/action.yaml

```diff
@@ -8,6 +8,7 @@ runs:
   using: composite
   steps:
   - name: Update plugin Version
+    shell: bash
```

Comment:
Out of curiosity, what's the behavior without the shell specification? Was the action failing?